### ISSUE TITLE
Define structure of tools

### ISSUE THREAD

**Issue by Greenheart** [Greenheart/idg.tools#29]
A tool consists of:

- name
- description
- Call to action - one or more ways to use this tool into practice. For example "1) Do this course, 2) do this course with others to get more value"
- IDG relevancy scores
- resources (further exploration and background)

We also discussed `level of scientific support`  that could use same relevancy score feature but different visual design. Ultimately we decided to use `Research and resources` instead to show the level of scientific support, and if we have few resource links for a tool, we show a disclaimer text with CTA to contribute with improved research and tool

Basically, the number and quality of linked resources will show the level of scientific support in a implicit and transparent way. It also removes the need to debate and maintain our own custom `scientific relevancy score`.

**Comment by chrismact** [comment]
https://www.figma.com/file/dj1eaMaWVKt18tfFengORu/better?node-id=218%3A1147

**Comment by Greenheart** [comment]
@chrismact for the `resources` to each tool, I think we want a markdown-formatted field where we encourage content co-creators to simply use the rich text bullet/ordered lists to add resources. This is much simpler for us and allows us to get started quickly.

Then in the future, we could update the `resources` to be a list of resources with a specific format. But a bullet list with links is simple and should be enough for a MVP.

**Comment by Greenheart** [comment]
Maybe use a peer-review step to validate the resources that are included?

Reliability when comparing multiple different reviewers.

Scientific advisory board might be able to create a transparent process to help review the quality of tools, and the skill relevancy scores as well as scientific support.

---

## Levels of support (quick suggestion)

1. new method (no evidence)
2. some evidence and used with possible positive effects
3. Some scientific support
4. Significant scientific support
5. Major scientific support - Meta analysis provided

Make it easy for people who want to perform studies. Link to ready-made forms that researchers can use. Resource bank to help.

Transparent grading and processes are important.

**Comment by Greenheart** [comment]
Closing since this is resolved at this point. We can always make updates later on.
